test/acquisition/test_analytic.py

```diff
@@ -916,8 +1001,18 @@ def _test_noisy_expected_imrpovement(self, dtype: torch.dtype) -> None:
         # testing gradient through exp of log computation
         exp_log_val.sum().backward()
         # testing that first gradient element coincides. The second is in the
-        # regime where the naive implementation looses accuracy.
-        atol = 2e-5 if dtype == torch.float32 else 1e-12
+        # regime where the naive implementation loses accuracy.
+
+        # Note - for some reason, when the RBFKernel is used instead of
```

Comment:
See my comment above, RBF generally gives rise to lower rank covariance matrices. Can you try lowering its lengthscale or spacing out the observations more? That should make it more likely that the RBF kernel gives rise to a well conditioned kernel matrix.